snuba/state/__init__.py

```diff
@@ -4,16 +4,27 @@
 import random
 import re
 import time
+from builtins import AssertionError
```

Comment:
Why do you need to import this?

Reply:
Oh I don't. The auto importer added it. I'll remove.